packages/quantic/cypress/integration/pager/pager.cypress.ts

```diff
@@ -0,0 +1,108 @@
+import {configure} from '../../page-objects/configurator';
+import {InterceptAliases, interceptSearch} from '../../page-objects/search';
+import {PagerExpectations as Expect} from './pager-expectations';
+import {PagerActions as Actions} from './pager-actions';
+import {stubConsoleError} from '../console-selectors';
+
+interface PagerOptions {
+  numberOfPages: number;
+}
+
+describe('quantic-pager', () => {
+  const pageUrl = 's/quantic-pager';
+
+  function visitPager(options: Partial<PagerOptions>, waitForSearch = true) {
+    interceptSearch();
+    cy.visit(pageUrl);
+    configure(options);
+    if (waitForSearch) {
+      cy.wait(InterceptAliases.Search);
+    }
+  }
+
+  function loadFromUrlHash(
+    options: Partial<PagerOptions> = {},
+    urlHash: string
+  ) {
+    interceptSearch();
+    cy.visit(`${pageUrl}#${urlHash}`);
+    configure(options);
+  }
+
+  function performSearchQuery() {
+    cy.get('.configurator__perform-search').click();
```

Comment:
should this be defined in a selector object? Are we likely to reuse this in other tests?

Reply:
I chose not to add a selector yet because the button is specific to this test suite. But if we make the "perform search" button reusable, then yes, we should provide a selector we could easily use.